### PR TITLE
Snow-map Salvage Project PR#1

### PR DESCRIPTION
**Pull request by ghost**
Adds snow gear cabinets:
A cabinet that has 5 winter boots and 5 winter coats in it

:cl:
 * tweak: Wooden cabinets cannot be welded anymore.

### CODE REVIEW

File: code/game/objects/structures/crates_lockers/closets/gimmick.dm [hunk 1]
```diff
@@ -4,12 +4,73 @@
...
+/obj/structure/closet/cabinet/snow/New()
+	..()
+	hanger_rail = new (src)
+
+/obj/structure/closet/cabinet/snow/close()
+	. = ..()
+	if(.)
+		unlock_atom(hanger_rail)
+		hanger_rail.forceMove(src)
+		return .
```
Comment (DamianX) [inline code/game/objects/structures/crates_lockers/closets/gimmick.dm]: delet this

File: code/game/objects/structures/crates_lockers/closets/gimmick.dm [hunk 2]
```diff
@@ -4,12 +4,73 @@
...
+
+/obj/structure/closet/cabinet/snow/open()
+	. = ..()
+	if(.)
+		hanger_rail.forceMove(get_turf(src))
+		lock_atom(hanger_rail)
+		return .
```
Comment (DamianX) [inline code/game/objects/structures/crates_lockers/closets/gimmick.dm]: delet this

File: code/game/objects/structures/crates_lockers/closets/gimmick.dm [hunk 3]
```diff
@@ -4,12 +4,73 @@
...
+	desc = "This rail is for holding NT standard-issue winter coats."
+	var/coats = 4
+	density = 0
+	anchored = 1
+	flags = FPRINT
+	layer = ABOVE_OBJ_LAYER
+
+/obj/structure/hanger_rail/attack_hand(var/mob/user)
+	..()
+	if(!Adjacent(user))
```
Comment (DamianX) [inline code/game/objects/structures/crates_lockers/closets/gimmick.dm]: `attack_hand`'s second parameter is `proximity` now, use that instead

**Review by DamianX — COMMENTED** [review]
also there doesn't seem to be a way to put the coats back on the hanger?

**Comment by ghost** [comment]
Fixed

**Comment by DamianX** [comment]
You say five but the code says four?
Also, added a changelog for you.

**Comment by ghost** [comment]
@DamianX Thanks. Yeah, there's four, I guess.

**Pull request merged**